src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -142,33 +79,16 @@ internal static unsafe bool TryReadFile(Stream stream, byte* buffer, long start,
             return true;
         }
 
-
-// The library guards against unavailable entrypoints by using EntryPointNotFoundException.
-#pragma warning disable BCL0015 // Diasable Pinvoke analyzer errors.
-        private static unsafe class NativeMethods
-        {
-            // API sets available on modern platforms:
-            [DllImport(@"api-ms-win-core-file-l1-1-0.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
-            [return: MarshalAs(UnmanagedType.Bool)]
-            internal static extern bool ReadFileModern(
-                 SafeHandle fileHandle,
-                 byte* buffer,
-                 int byteCount,
-                 out int bytesRead,
-                 IntPtr overlapped
-            );
-
-            // older Windows systems:
-            [DllImport(@"kernel32.dll", EntryPoint = "ReadFile", ExactSpelling = true, SetLastError = true)]
-            [return: MarshalAs(UnmanagedType.Bool)]
-            internal static extern bool ReadFileCompat(
-                 SafeHandle fileHandle,
-                 byte* buffer,
-                 int byteCount,
-                 out int bytesRead,
-                 IntPtr overlapped
-            );
-        }
+#pragma warning disable BCL0015 // Disable Pinvoke analyzer errors.
```

Comment:
I would prefer any BCL0015 be avoided and instead listed in PinvokeAnalyzerExceptionList.analyzerdata so they are in one place and not overlooked.

For ReadFile in particular it is already in both OneCore and UWP approved lists so this should not be necessary anyway.